tests/acceptance/setup/test.js

```diff
@@ -2,10 +2,10 @@ import {expect} from 'chai';
 import chai from 'chai';
 import Mugshot from 'mugshot';
 import chaiMugshot from 'chai-mugshot';
+import WebdriverIOAdapter from 'mugshot-webdriverio';
```

Comment:
This needs to be added to dev dependencies.

The tests actually failed:

![image](https://cloud.githubusercontent.com/assets/12442713/18317006/4df21f1e-7525-11e6-8d7d-4998ec095a75.png)

but we ignore the failure. The next tests passed, which shouldn't have. I think #48 addressed that, right @adela-istrate-hs ?

Regardless, we need to update the toolchain to support generating diffs when the assertion is negated.